label_studio/tasks/models.py

```diff
@@ -992,6 +992,44 @@ class Meta:
         db_table = 'prediction'
 
 
+class FailedPrediction(models.Model):
+    """
+    Class for storing failed prediction(s) for a task
+    """
+    message = models.TextField(
+        _('message'),
+        default='',
+        blank=True,
+        null=True,
+        help_text='The message explaining why generating this prediction failed'
+    )
+    error_type = models.CharField(
+        _('error_type'),
+        max_length=512,
+        default=None,
+        null=True,
+        help_text='The type of error that caused prediction to fail'
+    )
+    model_version = models.TextField(
+        _('model version'),
+        default='',
+        blank=True,
+        null=True,
+        help_text='A string value that for model version that produced the failed prediction. Used in both live models and when uploading offline predictions.',
+    )
+    model_run = models.ForeignKey(
```

Comment:
and in this case, how do we reference ML backend ?

Reply:
Discussed on Slack - will add a reference to ML backend model to FailedPrediction 
On delete will set to null instead of cascade 

Model run will not be set in case that we are using ML backend yes